include/solvers/heat_transfer_assemblers.h

```diff
@@ -105,8 +68,10 @@ class HeatTransferAssemblerCore : public HeatTransferAssemblerBase<dim>
    * @param copy_data (see base class)
    */
   virtual void
-  assemble_rhs(HeatTransferScratchData<dim> &scratch_data,
-               StabilizedMethodsCopyData    &copy_data) override;
+  assemble_rhs(const HeatTransferScratchData<dim> &scratch_data,
+               StabilizedMethodsCopyData          &copy_data) override;
+
+  const std::shared_ptr<SimulationControl> simulation_control;
```

Comment:
simulation_control used to be protected. I think here it should be either protected or private

Reply:
I'll change it back to protected, but I'm not sure why only here it was protected.